services/121-service/src/transaction-jobs/services/transaction-jobs-helper.service.ts

```diff
@@ -156,6 +161,7 @@ export class TransactionJobsHelperService {
     transaction.status = status;
     transaction.transactionStep = 1;
     transaction.errorMessage = errorMessage ?? null;
+    transaction.customData = customData ?? {};
```

Comment:
Nice, I also added customData to the function

Reply:
indeed ;)
<img width="737" height="348" alt="image" src="https://github.com/user-attachments/assets/be95e971-a8f1-4672-b4f6-3d62ffadf7ac" />